.ci/publish.sh

```diff
@@ -26,8 +26,6 @@ release/*)
     ;;
 esac
 
-bundle exec jekyll build
```

Comment:
build benutzt die _config.yml die oben geschrieben wird. das in 2 Skripte zu trennen ist unnötig und hier macht es sogar den build kaputt.